### ISSUE TITLE
Not OK: perl v5.6.0 on MSWin32-x86 4.0

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#1492]
Migrated from [rt.perl.org#2702](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=2702) (status was 'resolved')

Searchable as RT2702$

**Comment by p5pRT** [comment]
### From mchase@ix.netcom.com



Environment&#8203;:
&nbsp;   WinNT 4\.0 WS \(SP4\) using cmd\.exe as the shell
&nbsp;   gcc 2\.95\.2 from gcc\-2\.95\.2\-msvcrt\.exe from
http&#8203;://www\.xraylith\.wisc\.edu/~khan/software/gnu\-win32/
&nbsp;   dmake from dmake\-4\.1pl1\-win32\.zip from CPAN

Problem&#8203;:
I received the following error while building the Socket dynamic module
during the perl build\.  Socket was the first dynamic module attempted\.  This
bug report was generated by miniperl\.exe\.

It appears that the first argument to strNE\(\) called in XS\_VERSION\_BOOTCHECK
expects a different type of pointer than that provided in XS\_VERSION\.

Possible fixes&#8203;:
&nbsp;   1\.  Tell the compiler to accept this error\.
&nbsp;   2\.  Cast the first argument to strNE in XS\_VERSION\_BOOTCHECK to an
appropriate type\.
&nbsp;   3\.  Cast the first argument to strcmp\(\) on strNE to an appropriate type\.

makefile\.mk diff&#8203;:
<details><summary>Inline Patch</summary>

```diff
--- C:\install\old\perl-5.6.0\win32\makefile.mk Wed Mar 22 21:41:42 2000
+++ C:\install\new\perl-5.6.0\win32\makefile.mk Fri Mar 24 12:21:17 2000
@@ -165,7 +165,8 @@
 #
 #CCHOME  *= c:\bc5
 #CCHOME  *= $(MSVCDIR)
-CCHOME  *= c:\gcc-2.95.2-msvcrt
+#CCHOME  *= c:\gcc-2.95.2-msvcrt
+CCHOME  *= c:\gcc-2.95.2
 CCINCDIR *= $(CCHOME)\include
 CCLIBDIR *= $(CCHOME)\lib

@@ -210,7 +211,7 @@
 # set this to your email address (perl will guess a value from
 # from your loginname and your hostname, which may not be right)
 #
-#EMAIL  *=
+EMAIL  *=mchase@ix.netcom.com

 ##
 ## Build configuration ends.



```
</details>
Log Segment: (wrapped at 80 columns)
=========================================
cd \.\.\\ext\\Socket && \\
&nbsp;        \.\.\\\.\.\\miniperl \-I\.\.\\\.\.\\lib Makefile\.PL INSTALLDIRS=perl
Writing Makefile for Socket
cd \.\.\\ext\\Socket && c&#8203;:\\bin\\dmake\.exe \-S
mkdir \.\.\\\.\.\\lib\\auto\\Socket
cp Socket\.pm \.\.\\\.\.\\lib\\Socket\.pm
\.\.\\\.\.\\miniperl \-I\.\.\\\.\.\\lib \-I\.\.\\\.\.\\lib
\.\.\\\.\.\\lib\\ExtUtils/xsubpp \-noprototypes \-
typemap \.\.\\\.\.\\lib\\ExtUtils\\typemap Socket\.xs > Socket\.xsc &&
\.\.\\\.\.\\miniperl \-I\.\.
\\\.\.\\lib \-I\.\.\\\.\.\\lib \-MExtUtils&#8203;::Command \-e mv Socket\.xsc Socket\.c
gcc \-c  \-g \-O2 \-DWIN32 \-fno\-strict\-aliasing \-DPERL\_MSVCRT\_READFIX \-g \-O2
&nbsp;    \-DVERSION=\\"1\.72\\"  \-DXS\_VERSION=\\"1\.72\\"  \-I\.\.\\\.\.\\lib\\CORE  Socket\.c
Socket\.c&#8203;: In function \`boot\_Socket'&#8203;:
Socket\.c&#8203;:1224&#8203;: incompatible type for argument 1 of \`strcmp'
dmake\.exe&#8203;:  Error code 1\, while making 'Socket\.o'
dmake\.exe&#8203;:  Error code 255\, while making '\.\.\\lib\\auto\\Socket\\Socket\.dll'
==========================================

Several similar warnings appeared while building miniperl\.exe without
aborting the build\.  Here's one example&#8203;:
==========================================
gcc \-c \-I\.\.\\lib\\CORE \-I\.\\include \-I\. \-I\.\. \-DWIN32  \-DPERLDLL \-DPERL\_CORE \-g 
\-O2
\-DPERL\_EXTERNAL\_GLOB \-o\.\\mini\\doio\.o \.\.\\doio\.c
\.\.\\doio\.c&#8203;: In function \`Perl\_do\_aexec5'&#8203;:
\.\.\\doio\.c&#8203;:1287&#8203;: warning&#8203;: passing arg 2 of \`win32\_execvp' from incompatible
point
er type
\.\.\\doio\.c&#8203;:1289&#8203;: warning&#8203;: passing arg 2 of \`win32\_execvp' from incompatible
point
er type
==========================================

<details><summary>Perl Info</summary>

```

Flags:
    category=install
    severity=none

Site configuration information for perl v5.6.0:

Configured by xxChase at Fri Mar 24 10:22:30 2000.

Summary of my perl5 (revision 5 version 6 subversion 0) configuration:
  Platform:
    osname=MSWin32, osvers=4.0, archname=MSWin32-x86
    uname=''
    config_args='undef'
    hint=recommended, useposix=true, d_sigaction=undef
    usethreads=undef use5005threads=undef useithreads=undef
usemultiplicity=undef
    useperlio=undef d_sfio=undef uselargefiles=undef
    use64bitint=undef use64bitall=undef uselongdouble=undef usesocks=undef
  Compiler:
    cc='gcc', optimize='-g -O2', gccversion=
    cppflags='-DWIN32'
    ccflags ='-g -O2 -DWIN32  -fno-strict-aliasing -DPERL_MSVCRT_READFIX'
    stdchar='char', d_stdstdio=define, usevfork=false
    intsize=4, longsize=4, ptrsize=4, doublesize=8
    d_longlong=undef, longlongsize=8, d_longdbl=define, longdblsize=12
    ivtype='long', ivsize=4, nvtype='double', nvsize=8, Off_t='off_t',
lseeksize=4
    alignbytes=8, usemymalloc=n, prototype=define
  Linker and Libraries:
    ld='gcc', ldflags
=' -L"c:\perl\5.6.0\lib\MSWin32-x86\CORE" -L"c:\gcc-2.95.2\lib"'
    libpth=c:\gcc-2.95.2\lib




libs= -lmsvcrt -lmoldname -lkernel32 -luser32 -lgdi32 -lwinspool -lcomdlg32 
-ladvapi32 -lshell32 -lole32 -loleaut32 -lnetapi32 -luuid -lwsock32 -lmpr -l
winmm -lversion -lodbc32
    libc=-lmsvcrt, so=dll, useshrplib=yes, libperl=libperl56.a
  Dynamic Linking:
    dlsrc=dl_win32.xs, dlext=dll, d_dlsymun=undef, ccdlflags=' '
    cccdlflags=' ',
lddlflags='-mdll  -L"c:\perl\5.6.0\lib\MSWin32-x86\CORE" -L"c:\gcc-2.95.2\li
b"'

Locally applied patches:



@INC for perl v5.6.0:
    C:/install/new/perl-5.6.0/lib
    .


Environment for perl v5.6.0:
    HOME=C:\home\mchase
    LANG (unset)
    LANGUAGE (unset)
    LD_LIBRARY_PATH (unset)
    LOGDIR (unset)

PATH=c:\gcc-2.95.2\bin;C:\MSOFFICE\Office;c:\bin;c:\cygnus\cygwin~1\H-i586~1
\bin;c:\perl\5.00503\bin\MSWin32-x86;c:\perl\5.00503\bin;C:\WINNT\system32;C
:\WINNT;C:\Oracle\Ora81\bin;C:\Program Files\Oracle\jre\1.1.7\bin;C:\BC5\BIN
    PERL_BADLANG (unset)
    SHELL (unset)

Complete configuration data for perl v5.6.0:

Author=''
CONFIGDOTSH='true'
Date='$Date'
Header=''
Id='$Id'
Locker=''
Log='$Log'
Mcc='Mcc'
PATCHLEVEL='6'
PERL_API_REVISION='5'
PERL_API_SUBVERSION='0'
PERL_API_VERSION='5'
PERL_REVISION='5'
PERL_SUBVERSION='0'
PERL_VERSION='6'
RCSfile='$RCSfile'
Revision='$Revision'
SUBVERSION='0'
Source=''
State=''
_a='.a lib_ext=.a'
_exe='.exe'
_o='.o obj_ext=.o'
afs='false'
alignbytes='8'
ansi2knr=''
aphostname=''
api_revision='5'
api_subversion='0'
api_version='5'
api_versionstring='5.5.0'
ar='ar'
archlib='c:\perl\5.6.0\lib\MSWin32-x86'
archlibexp='c:\perl\5.6.0\lib\MSWin32-x86'
archname='MSWin32-x86'
archname64=''
archobjs=''
awk='awk'
baserev='5'
bash=''
bin='c:\perl\5.6.0\bin\MSWin32-x86'
bincompat5005=''
binexp='c:\perl\5.6.0\bin\MSWin32-x86'
bison=''
byacc='byacc'
byteorder='1234'
c=''
castflags='0'
cat='type'
cc='gcc'
cccdlflags=' '
ccdlflags=' '
ccflags='-g -O2 -DWIN32  -fno-strict-aliasing -DPERL_MSVCRT_READFIX'
ccsymbols=''
cf_by='xxChase'
cf_email='mchase@ix.netcom.com'
cf_time='Fri Mar 24 10:22:30 2000'
charsize='1'
chgrp=''
chmod=''
chown=''
clocktype='clock_t'
comm=''
compress=''
contains='grep'
cp='copy'
cpio=''
cpp='gcc -E'
cpp_stuff='42'
cppccsymbols=''
cppflags='-DWIN32'
cpplast=''
cppminus='-'
cpprun='gcc -E'
cppstdin='gcc -E'
cppsymbols=''
crosscompile=''
cryptlib=''
csh=''
d_Gconvert='sprintf((b),"%.*g",(n),(x))'
d_PRIEldbl=''
d_PRIFldbl=''
d_PRIGldbl=''
d_PRIX64=''
d_PRId64=''
d_PRIeldbl=''
d_PRIfldbl=''
d_PRIgldbl=''
d_PRIi64=''
d_PRIo64=''
d_PRIu64=''
d_PRIx64=''
d_access='define'
d_accessx=''
d_alarm=''
d_archlib='define'
d_atolf=''
d_atoll=''
d_attribut='define'
d_bcmp=''
d_bcopy=''
d_bincompat5005=''
d_bsd='define'
d_bsdgetpgrp=''
d_bsdsetpgrp=''
d_bzero=''
d_casti32='define'
d_castneg='define'
d_charvspr=''
d_chown=''
d_chroot=''
d_chsize='define'
d_closedir='define'
d_const='define'
d_crypt=''
d_csh=''
d_cuserid=''
d_dbl_dig='define'
d_difftime='define'
d_dirnamlen='define'
d_dlerror='define'
d_dlopen='define'
d_dlsymun=''
d_dosuid=''
d_drand48proto=''
d_dup2='define'
d_eaccess=''
d_endgrent=''
d_endhent=''
d_endnent=''
d_endpent=''
d_endpwent=''
d_endsent=''
d_endspent=''
d_eofnblk='define'
d_eunice=''
d_fchmod=''
d_fchown=''
d_fcntl=''
d_fd_macros='define'
d_fd_set='define'
d_fds_bits='define'
d_fgetpos='define'
d_flexfnam='define'
d_flock='define'
d_fork=''
d_fpathconf=''
d_fpos64_t=''
d_fs_data_s=''
d_fseeko=''
d_fsetpos='define'
d_fstatfs=''
d_fstatvfs=''
d_ftello=''
d_ftime='define'
d_getcwd=''
d_getfsstat=''
d_getgrent=''
d_getgrps=''
d_gethbyaddr='define'
d_gethbyname='define'
d_gethent=''
d_gethname='define'
d_gethostprotos='define'
d_getlogin='define'
d_getmnt=''
d_getmntent=''
d_getnbyaddr=''
d_getnbyname=''
d_getnent=''
d_getnetprotos=''
d_getpbyname='define'
d_getpbynumber='define'
d_getpent=''
d_getpgid=''
d_getpgrp=''
d_getpgrp2=''
d_getppid=''
d_getprior=''
d_getprotoprotos='define'
d_getpwent=''
d_getsbyname='define'
d_getsbyport='define'
d_getsent=''
d_getservprotos='define'
d_getspent=''
d_getspnam=''
d_gettimeod=''
d_gnulibc=''
d_grpasswd=''
d_hasmntopt=''
d_htonl='define'
d_iconv=''
d_index=''
d_inetaton=''
d_int64_t=''
d_isascii='define'
d_killpg=''
d_lchown=''
d_ldbl_dig='define'
d_link='define'
d_locconv='define'
d_lockf=''
d_longdbl='define'
d_longlong=''
d_lseekproto='define'
d_lstat=''
d_madvise=''
d_mblen='define'
d_mbstowcs='define'
d_mbtowc='define'
d_memchr='define'
d_memcmp='define'
d_memcpy='define'
d_memmove='define'
d_memset='define'
d_mkdir='define'
d_mkdtemp=''
d_mkfifo=''
d_mkstemp=''
d_mkstemps=''
d_mktime='define'
d_mmap=''
d_mprotect=''
d_msg=''
d_msg_ctrunc=''
d_msg_dontroute=''
d_msg_oob=''
d_msg_peek=''
d_msg_proxy=''
d_msgctl=''
d_msgget=''
d_msgrcv=''
d_msgsnd=''
d_msync=''
d_munmap=''
d_mymalloc=''
d_nice=''
d_nv_preserves_uv='define'
d_off64_t=''
d_old_pthread_create_joinable=''
d_oldpthreads=''
d_oldsock=''
d_open3=''
d_pathconf=''
d_pause='define'
d_phostname=''
d_pipe='define'
d_poll=''
d_portable='define'
d_pthread_yield=''
d_pwage=''
d_pwchange=''
d_pwclass=''
d_pwcomment=''
d_pwexpire=''
d_pwgecos=''
d_pwpasswd=''
d_pwquota=''
d_qgcvt=''
d_quad=''
d_readdir='define'
d_readlink=''
d_rename='define'
d_rewinddir='define'
d_rmdir='define'
d_safebcpy=''
d_safemcpy=''
d_sanemcmp='define'
d_sched_yield=''
d_scm_rights=''
d_seekdir='define'
d_select='define'
d_sem=''
d_semctl=''
d_semctl_semid_ds=''
d_semctl_semun=''
d_semget=''
d_semop=''
d_setegid=''
d_seteuid=''
d_setgrent=''
d_setgrps=''
d_sethent=''
d_setlinebuf=''
d_setlocale='define'
d_setnent=''
d_setpent=''
d_setpgid=''
d_setpgrp=''
d_setpgrp2=''
d_setprior=''
d_setpwent=''
d_setregid=''
d_setresgid=''
d_setresuid=''
d_setreuid=''
d_setrgid=''
d_setruid=''
d_setsent=''
d_setsid=''
d_setspent=''
d_setvbuf='define'
d_sfio=''
d_shm=''
d_shmat=''
d_shmatprototype=''
d_shmctl=''
d_shmdt=''
d_shmget=''
d_sigaction=''
d_sigsetjmp=''
d_socket='define'
d_socklen_t=''
d_sockpair=''
d_sqrtl=''
d_statblks=''
d_statfs_f_flags=''
d_statfs_s=''
d_statvfs=''
d_stdio_cnt_lval='define'
d_stdio_ptr_lval='define'
d_stdio_stream_array=''
d_stdiobase='define'
d_stdstdio='define'
d_strchr='define'
d_strcoll='define'
d_strctcpy='define'
d_strerrm='strerror(e)'
d_strerror='define'
d_strtod='define'
d_strtol='define'
d_strtold=''
d_strtoll=''
d_strtoul='define'
d_strtoull=''
d_strtouq=''
d_strxfrm='define'
d_suidsafe=''
d_symlink=''
d_syscall=''
d_sysconf=''
d_sysernlst=''
d_syserrlst='define'
d_system='define'
d_tcgetpgrp=''
d_tcsetpgrp=''
d_telldir='define'
d_telldirproto='define'
d_time='define'
d_times='define'
d_truncate=''
d_tzname='define'
d_umask='define'
d_uname='define'
d_union_semun='define'
d_ustat=''
d_vendorarch=''
d_vendorbin=''
d_vendorlib=''
d_vfork=''
d_void_closedir=''
d_voidsig='define'
d_voidtty=''
d_volatile='define'
d_vprintf='define'
d_wait4=''
d_waitpid='define'
d_wcstombs='define'
d_wctomb='define'
d_xenix=''
date='date'
db_hashtype='int'
db_prefixtype='int'
defvoidused='15'
direntrytype='struct direct'
dlext='dll'
dlsrc='dl_win32.xs'
doublesize='8'
drand01='(rand()/(double)((unsigned)1<<RANDBITS))'
dynamic_ext='Socket IO Fcntl Opcode SDBM_File POSIX attrs Thread B re
Data/Dumper Devel/Peek ByteLoader Devel/DProf File/Glob   Sys/Hostname'
eagain='EAGAIN'
ebcdic=''
echo='echo'
egrep='egrep'
emacs=''
eunicefix=':'
exe_ext='.exe'
expr='expr'
extensions='DynaLoader Socket IO Fcntl Opcode SDBM_File POSIX attrs Thread B
re   Data/Dumper Devel/Peek ByteLoader Devel/DProf File/Glob   Sys/Hostname
Errno'
fflushNULL='define'
fflushall=''
find='find'
firstmakefile='makefile'
flex=''
fpossize='4'
fpostype='fpos_t'
freetype='void'
full_ar=''
full_csh=''
full_sed=''
gccversion=''
gidformat='"ld"'
gidsign='-1'
gidsize='4'
gidtype='gid_t'
glibpth='/usr/shlib  /lib/pa1.1 /usr/lib/large /lib /usr/lib /usr/lib/386
/lib/386 /lib/large /usr/lib/small /lib/small /usr/ccs/lib /usr/ucblib
/usr/shlib '
grep='grep'
groupcat=''
groupstype='gid_t'
gzip='gzip'
h_fcntl='false'
h_sysfile='true'
hint='recommended'
hostcat='ypcat hosts'
huge=''
i16size='2'
i16type='short'
i32size='4'
i32type='long'
i64size='8'
i64type='long long'
i8size='1'
i8type='char'
i_arpainet='define'
i_bsdioctl=''
i_db=''
i_dbm=''
i_dirent='define'
i_dld=''
i_dlfcn='define'
i_fcntl='define'
i_float='define'
i_gdbm=''
i_grp=''
i_iconv=''
i_ieeefp=''
i_inttypes=''
i_limits='define'
i_locale='define'
i_machcthr=''
i_malloc='define'
i_math='define'
i_memory=''
i_mntent=''
i_ndbm=''
i_netdb=''
i_neterrno=''
i_netinettcp=''
i_niin=''
i_poll=''
i_pthread=''
i_pwd=''
i_rpcsvcdbm='define'
i_sfio=''
i_sgtty=''
i_shadow=''
i_socks=''
i_stdarg='define'
i_stddef='define'
i_stdlib='define'
i_string='define'
i_sunmath=''
i_sysaccess=''
i_sysdir=''
i_sysfile=''
i_sysfilio='define'
i_sysin=''
i_sysioctl=''
i_syslog=''
i_sysmman=''
i_sysmode=''
i_sysmount=''
i_sysndir=''
i_sysparam=''
i_sysresrc=''
i_syssecrt=''
i_sysselct=''
i_syssockio=''
i_sysstat='define'
i_sysstatfs=''
i_sysstatvfs=''
i_systime=''
i_systimek=''
i_systimes=''
i_systypes='define'
i_sysuio=''
i_sysun=''
i_sysutsname=''
i_sysvfs=''
i_syswait=''
i_termio=''
i_termios=''
i_time='define'
i_unistd=''
i_ustat=''
i_utime='define'
i_values=''
i_varargs=''
i_varhdr='varargs.h'
i_vfork=''
ignore_versioned_solibs=''
inc_version_list=''
inc_version_list_init='0'
incpath='c:\gcc-2.95.2\include'
inews=''
installarchlib='c:\perl\5.6.0\lib\MSWin32-x86'
installbin='c:\perl\5.6.0\bin\MSWin32-x86'
installhtmldir='c:\perl\5.6.0\html'
installhtmlhelpdir='c:\perl\5.6.0\htmlhelp'
installman1dir='c:\perl\5.6.0\man\man1'
installman3dir='c:\perl\5.6.0\man\man3'
installprefix='c:\perl\5.6.0'
installprefixexp='c:\perl\5.6.0'
installprivlib='c:\perl\5.6.0\lib'
installscript='c:\perl\5.6.0\bin'
installsitearch='c:\perl\site\5.6.0\lib\MSWin32-x86'
installsitebin='c:\perl\5.6.0\bin\MSWin32-x86'
installsitelib='c:\perl\site\5.6.0\lib'
installstyle='lib'
installusrbinperl=''
installvendorarch=''
installvendorbin=''
installvendorlib=''
intsize='4'
ivdformat='"ld"'
ivsize='4'
ivtype='long'
known_extensions='DynaLoader Socket IO Fcntl Opcode SDBM_File POSIX attrs
Thread B re   Data/Dumper Devel/Peek ByteLoader Devel/DProf File/Glob
Sys/Hostname Errno'
ksh=''
large=''
ld='gcc'
lddlflags='-mdll  -L"c:\perl\5.6.0\lib\MSWin32-x86\CORE" -L"c:\gcc-2.95.2\li
b"'
ldflags=' -L"c:\perl\5.6.0\lib\MSWin32-x86\CORE" -L"c:\gcc-2.95.2\lib"'
ldlibpthname=''
less='less'
lib_ext='.a'
libc='-lmsvcrt'
libperl='libperl56.a'
libpth='c:\gcc-2.95.2\lib'
libs=' -lmsvcrt -lmoldname -lkernel32 -luser32 -lgdi32 -lwinspool -lcomdlg32
 -ladvapi32 -lshell32 -lole32 -loleaut32 -lnetapi32 -luuid -lwsock32 -lmpr -
lwinmm -lversion -lodbc32'
libsdirs=''
libsfiles=''
libsfound=''
libspath=''
libswanted='net socket inet nsl nm ndbm gdbm dbm db malloc dl dld ld sun m c
cposix posix ndir dir crypt ucb bsd BSD PW x'
line='line'
lint=''
lkflags=''
ln=''
lns='copy'
locincpth='/usr/local/include /opt/local/include /usr/gnu/include
/opt/gnu/include /usr/GNU/include /opt/GNU/include'
loclibpth='/usr/local/lib /opt/local/lib /usr/gnu/lib /opt/gnu/lib
/usr/GNU/lib /opt/GNU/lib'
longdblsize='12'
longlongsize='8'
longsize='4'
lp=''
lpr=''
ls='dir'
lseeksize='4'
lseektype='off_t'
mail=''
mailx=''
make='dmake'
make_set_make='#'
mallocobj='malloc.o'
mallocsrc='malloc.c'
malloctype='void *'
man1dir='c:\perl\5.6.0\man\man1'
man1direxp='c:\perl\5.6.0\man\man1'
man1ext='1'
man3dir='c:\perl\5.6.0\man\man3'
man3direxp='c:\perl\5.6.0\man\man3'
man3ext='3'
medium=''
mips_type=''
mkdir='mkdir'
mmaptype='void *'
models='none'
modetype='mode_t'
more='more /e'
multiarch=''
mv=''
myarchname='MSWin32'
mydomain=''
myhostname=''
myuname=''
n='-n'
netdb_hlen_type='int'
netdb_host_type='char *'
netdb_name_type='char *'
netdb_net_type='long'
nm='nm'
nm_opt=''
nm_so_opt=''
nonxs_ext='Errno'
nroff=''
nvsize='8'
nvtype='double'
o_nonblock='O_NONBLOCK'
obj_ext='.o'
old_pthread_create_joinable=''
optimize='-g -O2'
orderlib='false'
osname='MSWin32'
osvers='4.0'
package='perl5'
pager='more /e'
passcat=''
patchlevel=''
path_sep=';'
perl='perl'
perl5=''
perladmin=''
perlpath='c:\perl\5.6.0\bin\MSWin32-x86\perl.exe'
pg=''
phostname='hostname'
pidtype='int'
plibpth=''
pm_apiversion='5.005'
pmake=''
pr=''
prefix='c:\perl'
prefixexp='c:'
privlib='c:\perl\5.6.0\lib'
privlibexp='c:\perl\5.6.0\lib'
prototype='define'
ptrsize='4'
quadkind='5'
quadtype='long long'
randbits='15'
randfunc='rand'
randseedtype='unsigned'
ranlib='rem'
rd_nodata='-1'
revision='5'
rm='del'
rmail=''
runnm='true'
sPRIEldbl='"E"'
sPRIFldbl='"F"'
sPRIGldbl='"G"'
sPRIX64='"lX"'
sPRId64='"ld"'
sPRIeldbl='"e"'
sPRIfldbl='"f"'
sPRIgldbl='"g"'
sPRIi64='"li"'
sPRIo64='"lo"'
sPRIu64='"lu"'
sPRIx64='"lx"'
sched_yield=''
scriptdir='c:\perl\5.6.0\bin'
scriptdirexp='c:\perl\5.6.0\bin'
sed='sed'
seedfunc='srand'
selectminbits='32'
selecttype='Perl_fd_set *'
sendmail='blat'
sh='cmd /x /c'
shar=''
sharpbang='#!'
shmattype='void *'
shortsize='2'
shrpenv=''
shsharp='true'
sig_count='26'
sig_name='ZERO NUM01 INT QUIT ILL NUM05 NUM06 NUM07 FPE KILL NUM10 SEGV
NUM12 PIPE ALRM TERM NUM16 NUM17 NUM18 NUM19 CHLD BREAK ABRT STOP NUM24 CONT
CLD'
sig_name_init='"ZERO", "NUM01", "INT", "QUIT", "ILL", "NUM05", "NUM06",
"NUM07", "FPE", "KILL", "NUM10", "SEGV", "NUM12", "PIPE", "ALRM", "TERM",
"NUM16", "NUM17", "NUM18", "NUM19", "CHLD", "BREAK", "ABRT", "STOP",
"NUM24", "CONT", "CLD", 0'
sig_num='0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25
20 0'
sig_num_init='0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17,
18, 19, 20, 21, 22, 23, 24, 25, 20, 0'
signal_t='void'
sitearch='c:\perl\site\5.6.0\lib\MSWin32-x86'
sitearchexp='c:\perl\site\5.6.0\lib\MSWin32-x86'
sitebin='c:\perl\site\5.6.0\bin\MSWin32-x86'
sitebinexp='c:\perl\site\5.6.0\bin\MSWin32-x86'
sitelib='c:\perl\site\5.6.0\lib'
sitelib_stem=''
sitelibexp='c:\perl\site\5.6.0\lib'
siteprefix='c:\perl\site\5.6.0'
siteprefixexp='c:\perl\site\5.6.0'
sizesize='4'
sizetype='size_t'
sleep=''
smail=''
small=''
so='dll'
sockethdr=''
socketlib=''
socksizetype='int'
sort='sort'
spackage='Perl5'
spitshell=''
split=''
src=''
ssizetype='int'
startperl='#!perl'
startsh='#!/bin/sh'
static_ext='DynaLoader'
stdchar='char'
stdio_base='((fp)->_base)'
stdio_bufsiz='((fp)->_cnt + (fp)->_ptr - (fp)->_base)'
stdio_cnt='((fp)->_cnt)'
stdio_filbuf=''
stdio_ptr='((fp)->_ptr)'
stdio_stream_array=''
strings='/usr/include/string.h'
submit=''
subversion=''
sysman='/usr/man/man1'
tail=''
tar=''
tbl=''
tee=''
test=''
timeincl='/usr/include/sys/time.h '
timetype='time_t'
touch='touch'
tr=''
trnl='\012'
troff=''
u16size='2'
u16type='unsigned short'
u32size='4'
u32type='unsigned long'
u64size='8'
u64type='unsigned long long'
u8size='1'
u8type='unsigned char'
uidformat='"ld"'
uidsign='-1'
uidsize='4'
uidtype='uid_t'
uname='uname'
uniq='uniq'
uquadtype='unsigned long long'
use5005threads=''
use64bitall=''
use64bitint=''
usedl='define'
useithreads=''
uselargefiles=''
uselongdouble=''
usemorebits=''
usemultiplicity=''
usemymalloc='n'
usenm='false'
useopcode='true'
useperlio=''
useposix='true'
usesfio='false'
useshrplib='yes'
usesocks=''
usethreads=''
usevendorprefix=''
usevfork='false'
usrinc='/usr/include'
uuname=''
uvoformat='"lo"'
uvsize='4'
uvtype='unsigned long'
uvuformat='"lu"'
uvxformat='"lx"'
vendorarch=''
vendorarchexp=''
vendorbin=''
vendorbinexp=''
vendorlib=''
vendorlib_stem=''
vendorlibexp=''
vendorprefix=''
vendorprefixexp=''
version='5.6.0'
vi=''
voidflags='15'
xlibpth='/usr/lib/386 /lib/386'
xs_apiversion='5.6.0'
zcat=''
zip='zip'




```
</details>

**Comment by p5pRT** [comment]
### From [Unknown Contact. See original ticket]

This is a duplicate of \[ID 20000324\.054\]\.  Please close it\.
